src/lib/registration/types.ts

```diff
@@ -0,0 +1,81 @@
+export interface IRegistration {
```

Comment:
Interfaces in here are pretty much taken from the `lib/types` file and put into an interface that represents a `Registration`.  I haven't bothered with worrying about registering multiple beacons yet.

I know this is the design in the API that @zackads was working on but we can get around this by creating a `serialise` method on the `Registration` class that acts as the translation from what the web app captures as a registration into what the backend API expects to receive. It should be easy enough to update this interface and the `Registration` class to handle multiple beacon registrations when that requirement comes along further down the line

@zackads thoughts?